.lintstagedrc.json

```diff
@@ -1,4 +1,4 @@
 {
   "*.js": "eslint --ext .js,.mjs",
-  "*.{scss,md,json,prisma}": "prettier --check"
-}
+  "*.{scss,md,json}": "prettier --check"
```

Comment:
Хаскі не вміє працювати із "prisma", та видає дає помилку, ПОЗБАВЛЯЄ можливості відправити свій код до github. 
I без цих змін, я \ чи будь хто, хто працює на операційній системі віндовс, змушений видаляти "prisma" із правил, відправляти код і тоді вертає назад, це правило.  Такий костиль, нам без сенсу підтримувати.

"Husky doesn't work with Prisma and throws an error, preventing the ability to push the code to GitHub. Without these changes, I, or anyone working on the Windows operating system, have to exclude 'Prisma' from the rules, push the code, and then revert this rule. This workaround makes no sense for us to maintain."

Reply:
true about workaround

altho deleting a prisma file from linting is not an option too, as that leads to degradating codebase ( in this case, db schema )

i suggest we ( or anybody else ) try to pair on that, and fix the actual issue with editor on windows, rather than changing part of codebase in favor of editor which is not setup correctly 

if nothing can be done, i would expect an explanation of that somewhere in the docs 🤷🏻‍♂️

Reply:
You're right. I will return the rule with a prism.